### PR TITLE
Create Audiocontroller with singleton pattern < feature, interface

### PR DESCRIPTION
**Pull request by spiderbiggen**

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
The implementation looks good, but resolve the build failures first please.

**Comment by SytzeAndr** [comment]
also: merge conflicts.

File: src/main/java/lasersharks/Options.java [hunk 1]
```diff
@@ -20,10 +20,23 @@
   private static final int DEFAULT_WIDTH = 1920;
   private static final int DEFAULT_HEIGHT = 1080;
   private static final String DEFAULT_MUSIC_FILENAME = "src/main/resources/music.mp3";
+  private static final String DEFAULT_HIT_SOUND_FILENAME = "src/main/resources/soundEffect1.wav";
```
Comment (Exclaminator) [inline src/main/java/lasersharks/Options.java]: Hit sound for what, please be more specific.

File: src/main/java/lasersharks/Options.java [hunk 2]
```diff
@@ -36,9 +49,22 @@
    *          the resolution this Options object should use.
    */
   public Options(Dimension screenRes) {
```
Comment (Exclaminator) [inline src/main/java/lasersharks/Options.java]: Please make this private or protected.

File: src/main/java/lasersharks/behaviour/eaten/FishEatenBehaviour.java [hunk 3]
```diff
@@ -26,6 +28,6 @@ public FishEatenBehaviour(SeaObject ele) {
   public void onCollisionEaten() {
     element.kill();
     element.setSize(0);
-    StandardPane.playSoundEffect(EAT_FISH_SOUND);
+    AudioController.getInstance().playSoundEffect(Options.getInstance().getHitSoundFileName());
```
Comment (Exclaminator) [inline src/main/java/lasersharks/behaviour/eaten/FishEatenBehaviour.java]: Why don't you have different playSound effect functions in your adio controller wich in turn fetch from the options menu? this structure is the beginning of god behaviour.

File: src/main/java/lasersharks/controllers/AudioController.java [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  private static AudioController instance;
+  
+
+  private MediaPlayer musicPlayer;
+  private MediaPlayer sfxPlayer;
+
+  /**
+   * 
+   */
+  public AudioController() {
```
Comment (Exclaminator) [inline src/main/java/lasersharks/controllers/AudioController.java]: please make this private or protected,

**Comment by Exclaminator** [comment]
PLease solve the merge conficts and please look at my comments.

File: src/main/java/lasersharks/Options.java (hunk 2 above)
Comment (Exclaminator) [inline src/main/java/lasersharks/Options.java]: Please protect this method.

**Comment by Exclaminator** [comment]
One comment, the rest looks fine!

**Comment by MichielvdBerg** [comment]
I approve

**Pull request merged**